swailing/logger.py

```diff
@@ -34,6 +41,13 @@ def __init__(self, name_or_logger, fill_rate=None, capacity=None, verbosity=HINT
         Setting it to DETAIL will stop hint outputs. Setting it to
         HINT allows all output.
 
+        When structured_detail is True, FancyLogContext.detail(msg) expects
```

Comment:
Say `logger.detail` instead of `FancyLogContext`, because a user looking at the docstring will have no idea what that is. It's an internal name, which is why I was tongue in cheek about it.